.gitignore

```diff
@@ -16,6 +16,7 @@ icu4c/lib32uwp
 icu4c/lib64
 icu4c/lib64uwp
 icu4c/libARMuwp
+icu4c/source/.vscode/*
```

Comment:
You don't need the `/*` in .gitignore syntax.  If you give a directory path, the whole directory is ignored.

Reply:
Actually, a better solution is to just put `.vscode/` at the top level.  That ignores all `.vscode` directories anywhere in the directory tree.

Reply:
Done :)

Thanks for the note.